src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/dto/OffenderBooking.kt

```diff
@@ -38,5 +38,6 @@ data class OffenderBooking (
   val recall: Boolean? = null,
   val imprisonmentStatus: String? = null,
   val personalCareNeeds : List<PersonalCareNeed>? = null,
-  val receptionDate: LocalDate? = null
+  val receptionDate: LocalDate? = null,
+  val locationDescription: String? = null
```

Comment:
I think with kotlin these days you can add in a trailing comma, will sort out issues like this where you've changed two lines when only one needed to be changed

Reply:
although it would be much nicer for the code formatter to do this for us like `prettier` does for javascript :thinking: